.github/workflows/test.yml

```diff
@@ -62,3 +62,5 @@ jobs:
         run: bundler exec rubocop --parallel
       - name: Lint Javascript and Sass files
         run: npm run lint
+      - name: Lint locale files
+        run: bundle exec i18n-tasks check-normalized
```

Comment:
🤔  I dont think we need this because we already do this as part of running rspec, https://github.com/alphagov/forms-admin/blob/main/spec/i18n_spec.rb

Reply:
Oh right I didn't realise that, thanks!

Reply:
I've taken that out and added a note about RSpec checking the locale files to the README.